test/Definition/RuntimeDefinitionTest.php

```diff
@@ -40,7 +40,7 @@ public function testSetExplicitClassesViaConstructor()
         $this->assertEquals($expected, $definition->getClasses());
     }
 
-    public function testSetExplicitClassesReplacesPrefiousValues()
+    public function testSetExplicitClassesReplacesPreviousValues()
```

Comment:
Probably `: void` to be added? 🤔

Reply:
I'm thinking of taking all the test class related changes to a separate branch and create a separate PR to update it in its entirety if that is okay with you?

Reply:
Works for me - also OK to leave it be, for now, as `laminas/laminas-coding-standard` should be capable of applying these in future :thinking: